src/uu/tail/src/tail.rs

```diff
@@ -1650,8 +1664,9 @@ impl PathExtTail for Path {
 
     /// Return true if `path` is is a file type that can be tailed
     fn is_tailable(&self) -> bool {
-        self.is_file() || self.exists() && self.metadata().unwrap().is_tailable()
+        self.is_file() || self.exists() && self.metadata().map_or(false, |meta| meta.is_tailable())
```

Comment:
The call to `metadata()` will return an error if user lacks permissions or the path does not exist. However, `is_file()` and `exists()` already return false if user cannot access metadata.  So this `unwrap` should never be triggered.

Reply:
But that's what happened

Reply:
Then there could be an issue with time-of-check / time-of-use. You could try `try_exists()` instead of `exists()`, but it's only available in rust 1.63

Reply:
Why do you want to stick to unwrap()?

Reply:
Silencing this error doesn't feel right.
This unwrap triggers, but (according to documentation) it shouldn't, which means there's an issue somewhere up in the call stack.

Reply:
It's not about silencing the error but instead returning such a path as untailable, that's what the signature and name of the method states. No surprises.

Reply:
Why not 
```rust
    self.metadata().map_or(false, |meta| meta.is_tailable())
```

Reply:
Good point